docs/06-actors-actions.md

```diff
@@ -0,0 +1,414 @@
+---
+title: Actors
+path: /docs/actors
+---
+
+## Basic actors
+
+For quick and dirty games, you can just create an instance of an [[Actor]]
+and manipulate it directly.
+
+Actors (and other entities) must be added to a [Scene](/docs/scene) to be drawn
+and updated on-screen.
+
+```ts
+const player = new ex.Actor()
+
+// move the player
+player.vel.x = 5
+
+// add player to the current scene
+game.add(player)
+```
+
+[[Engine.add|game.add]] is a convenience method for adding an actor to the current scene. The equivalent verbose call is [[Scene.add|game.currentScene.add]].
+
+## Actor Lifecycle
+
+An actor has a basic lifecycle that dictates how it is initialized, updated, and drawn. Once an actor is part of a
+[scene](/docs/scene), it will follow this lifecycle.
+
+![Actor Lifecycle](/assets/images/docs/ActorLifecycle.png)
+
+## Extending actors
+
+For "real-world" games, you'll want to extend the `Actor` class.
+This gives you much greater control and encapsulates logic for that
+actor.
+
+You can override the [[Actor.onInitialize]] method to perform any startup logic
+for an actor (such as configuring state). `onInitialize` gets called
+**once** before the first frame an actor is drawn/updated. It is passed
+an instance of [Engine](/docs/intro) to access global state or perform coordinate math.
```

Comment:
@kamranayub We may want to offer some motivation to use `onInitialize` over constructor time work. It allows actors to be lazily initialized before they are used, versus paying all the costs at constructor time (and potentially you specific game state is not yet at a useful point)

Reply:
What do you think about adding a custom element to denote "recommended usage"? To make it really apparent, similar to how I specified Advanced usage for onPost hooks

Reply:
That's a great idea